### ISSUE TITLE
Dark mode makes input field placeholders almost unreadable

### ISSUE THREAD

**Issue by nickater** [mackentish/BudgeIt#1]
<img src="https://github.com/mackentish/BudgeIt/assets/30247698/df98361c-654e-4ba4-b912-9b16cfd377fe" width="300" />

**Comment by mackentish** [comment]
Good catch, the app currently doesn't support dark mode so I need to update it by hardcoding the placeholderTextColor. I'll make a note of that
